design/drawer/item.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { classNames } from '@oacore/design/lib/utils'
+
+import styles from './styles.css'
+
+const FlatItem = ({ children, className, tag: Tag = 'a', ...restProps }) => (
+  <Tag className={classNames.use(styles.item).join(className)} {...restProps}>
+    {children}
+  </Tag>
+)
+
+const TreeItem = ({ children, ...passProps }) => (
+  <li>
+    <FlatItem {...passProps}>{children}</FlatItem>
+  </li>
+)
+
+const Item = ({ children, tag = 'li', ...passProps }) => {
```

Comment:
Interesting sometimes you call it passProps and sometimes restProps? Is there any difference?

Reply:
'rest' if I actually destructurise, pass if I just forward without destructuring.

This particular case is special, I destrucutrise to make correct forwarding.

Reply:
Maybe I should always call it `forwardProps`. What do you think?

Reply:
forwardProps sounds better but passProps is good too